vsphere/datadog_checks/vsphere/config.py

```diff
@@ -67,7 +77,7 @@ def __init__(self, instance, log):
             self.collected_resource_types = REALTIME_RESOURCES
 
         # Filters
-        self.resource_filters = self._parse_resource_filters(instance.get("resource_filters", {}))
+        self.resource_filters = self._parse_resource_filters(instance.get("resource_filters", []))
```

Comment:
Is this a discovered bug? 😄

Reply:
Yes and no, since it's doesn't cause any bug per se, but it's not the right type.